airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -82,7 +82,6 @@ definitions:
           - stream_interval
           - stream_partition
           - stream_slice
-          - stream_state
```

Comment:
I noticed in the file that we also reference `stream_state` under the interpolation variables. Let's get rid of this part for good as well:

https://github.com/airbytehq/airbyte-python-cdk/blob/4a1c4c91b3d6e6e2b158313e579290ecb210cbb6/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L3683-L3688